lib/screens/app/app.dart

```diff
@@ -24,7 +33,12 @@ class NavigationTab {
   final Function screenBuilder;
   final int index;
 
-  NavigationTab({this.title, this.icon, this.iconSelected, this.screenBuilder, this.index});
```

Comment:
your IDE is set to wrap hard at less than 120 which is the values we are using for the app code base. 
![image](https://user-images.githubusercontent.com/8041901/108429115-8ebfae00-720d-11eb-8356-29a5d83f554c.png)


This is causing unnecessary formatting changes. 

Please fix this, revert the formatting changes, then reopen the PR.

Reply:
@7flash please set your IDE to 120 width

Otherwise we keep wasting endless hours on these formatting changes...